package/gargoyle/files/www/js/basic.js

```diff
@@ -104,7 +104,12 @@ function saveChanges()
 			{
 				uci.set("wireless", wifiDevG, "htmode",  getSelectedValue("wifi_channel_width") );						//always set htmode, even if it is "NONE"
 			}
-			txPowerSet("wifi_max_txpower", "wifi_txpower", wifiDevG)
+			txPowerSet("wifi_max_txpower", "wifi_txpower", wifiDevG);
+			if (getSelectedValue("wifi_hwmode_2ghz") != "11g")
+			{
+				uci.set("wireless", wifiDevG, "require_mode", "n");
+				uci.set("wireless", wifiDevG, "greenfield", "1");
```

Comment:
require_mode just rejects auth&assoc requests, greenfield drops the traffic as noise)